Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -659,12 +670,28 @@ public final class PackageBuilder {
                     }
                 }
             } ?? []
-
-            // Create the target.
+            
+            // Get dependencies from the plugin usages of this target.
+            let pluginUsages: [Target.PluginUsage] = manifestTarget?.pluginUsages.map {
+                $0.compactMap{ usage in
+                    switch usage {
+                    case .plugin(let name, let package):
+                        if let package = package {
+                            return .product(Target.ProductReference(name: name, package: package), conditions: [])
+                        }
+                        else {
+                            guard let target = targets[name] else { return nil }
```

Comment:
should this be case-sensitive or case-insensitive?

Reply:
Same comment as the other one.